packages/salesforcedx-vscode-apex/test/vscode-integration/apexLspJarContents.test.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// tslint:disable:no-unused-expression
```

Comment:
Is `no-unused-expression` needed?  Otherwise it looks good.

Reply:
Not required. I copied the copyright text and part of the test and this came with it. Will remove.

Reply:
Actually turns out it is required. If not, lint is not happy with line 25 (complains "unused expression, expected an assignment or function call")

Reply:
you can remove this and use 
```expect(
      permGuardedClasses.some(permGuardedClass => stdout.includes(permGuardedClass))
    ).to.equal(false);```
either that or we should figure out how to disable this rule for the tests.

Reply:
holy moley we have this comment in our source ~110 times.  I'll just add a comment to the eslint ticket to ensure we just disable that rule and remove the comment for the eslint migration